### PR TITLE
bump sdk

### PR DESCRIPTION
**Pull request by liranr23**

### CODE REVIEW

**Comment by ahadas** [comment]
@liranr23 isn't it part of #713 ?

**Comment by liranr23** [comment]
> @liranr23 isn't it part of #713 ?

i'll close #713 . #713 was a try to bump not only the sdk, but also go and the ansible sdk. here it's about the operator sdk.

File: Makefile [hunk 1]
```diff
@@ -352,7 +352,7 @@ ifeq (, $(shell command -v opa))
 	@{ \
 	set -e ;\
 	mkdir -p ${HOME}/.local/bin ; \
-	curl -sL -o ${HOME}/.local/bin/opa https://openpolicyagent.org/downloads/v0.57.1/opa_linux_amd64_static ; \
+	curl -sL -o ${HOME}/.local/bin/opa https://openpolicyagent.org/downloads/v0.62.1/opa_linux_amd64_static ; \
```
Comment (ahadas) [inline Makefile:355]: that's unrelated to the SDK, right? I also wonder, why 0.62.1 while there's 0.65.0?
I'd suggest to separate this out to a different PR

**Review by ahadas — COMMENTED** [review]
one comment about OPA which doesn't seem related to this PR but let me know if I'm missing why it is related..

File: Makefile (hunk 1 above)
Comment (liranr23) [reply Makefile:355]: yeah that just alignment with bazel where we use 0.62...i'll make another PR for OPA

**Review by ahadas — APPROVED** [review]
@liranr23 do we want this on the release-v2.6.3 branch?

**Comment by liranr23** [comment]
> @liranr23 do we want this on the release-v2.6.3 branch?

yeah, let's backport it
